src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/Simulators/SimulatorsCommand.cs

```diff
@@ -180,7 +189,8 @@ protected async Task<IEnumerable<Simulator>> GetAvailableSimulators()
                 source: source,
                 installPrefix: installPrefix,
                 fileSize: (long)parsedFileSize,
-                isCryptexDiskImage: isCryptexDiskImage
+                isCryptexDiskImage: isCryptexDiskImage,
```

Comment:
Do we still need to logic for `isCryptexDiskImage` ?

Reply:
I think we can safely remove this now